internal/gps/result.go

```diff
@@ -51,11 +52,13 @@ type solution struct {
 // It requires a SourceManager to do the work, and takes a flag indicating
 // whether or not to strip vendor directories contained in the exported
 // dependencies.
-func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool) error {
+func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool, logger *log.Logger) error {
 	if l == nil {
 		return fmt.Errorf("must provide non-nil Lock to WriteDepTree")
 	}
 
+	logger.Println("Creating vendor dir ...")
```

Comment:
i don't think this message adds much value. for one, this func isn't actually guaranteed to be creating a `vendor/` dir - the way we use it right now, it's actually writing to a temporary directory, which will later be swapped into place as a `vendor/` dir, assuming no errors occur.

if dep wants to put some kind of header message like this in place from within `SafeWriter.Write()`, that's fine, but let's keep what's printed directly here to a minimum.